products/compute/api.yaml

```diff
@@ -14437,11 +14434,10 @@ objects:
                           - :TEMPORARY_REDIRECT
                       - !ruby/object:Api::Type::Boolean
                         name: 'stripQuery'
-                        default_value: false
```

Comment:
Can you add these back so that ansible still has them?

Reply:
Unfortunately not?

If we have a default value specified in api.yaml we cannot override it with a `nil` value that I know of

Reply:
Oh, but I can add them to ansible.yaml